features/obituaryGuestBook.feature

```diff
@@ -0,0 +1,14 @@
+@currentTest
+
+Feature: Janus Obituary Guestbook
+    In order to pay my respects
+    As a user
+    I want to be able to add a message to a guestbook
+
+    Scenario: Add message
+        Given I am on a Janus obituary page
+        When I scroll to the Guestbook section
+        Then I should be able to fill out the guestbook
+        And I should be able to submit the entry
```

Comment:
This also feels like an action. Maybe the assertion is something like `Then I should see my guestbook entry in the screener review queue`